### ISSUE TITLE
error in tutorial: lp.draw non-native

### ISSUE THREAD

**Issue by volkansevim** [harmslab/latticeproteins#3]
Visualization of non-native states fail for some sequences such as ['I', 'E', 'Q', 'Y', 'K', 'C', 'T', 'A', 'F', 'L']:

```
# Find the 5 lowest energy conformations.
alt_conf = lattice.confs.k_lowest_confs(seq, 1.0, 5)

# Choose the 5th lowest as the target fold.
target = alt_conf[-1]
lp.draw.in_notebook(seq, target)
```

```
KeyError                                  Traceback (most recent call last)
~/Software/latticeproteins/latticeproteins/draw.py in _add_item(self, x, y)
    194         try:
--> 195             if self.mapping[self.array[y][x]] is not None:
    196                 # Get SVG element to add

KeyError: 'F'

During handling of the above exception, another exception occurred:

IndexError                                Traceback (most recent call last)
<ipython-input-96-5b4dea7d1758> in <module>()
      4 # Choose the 5th lowest as the target fold.
      5 target = alt_conf[-1]
----> 6 lp.draw.in_notebook(seq, target)

~/Software/latticeproteins/latticeproteins/draw.py in in_notebook(sequence, conf, **kwargs)
     39 def in_notebook(sequence, conf, **kwargs):
     40     """Creates a Python SVG configuration object. Automagically displays in notebook."""
---> 41     drawing = Configuration(sequence, conf, **kwargs)
     42     return drawing
     43 

~/Software/latticeproteins/latticeproteins/draw.py in __init__(self, sequence, configuration, color_sequence, rotation, font_size, dot_scale, font_weight)
    156             self.color_sequence = color_sequence
    157         # Sets rotation and configuration
--> 158         self.rotate(rotation)
    159 
    160     @property

~/Software/latticeproteins/latticeproteins/draw.py in rotate(self, rotation)
    182         for i in range(n):
    183             self.configuration = "".join([ROTATE[c] for c in self.configuration])
--> 184         self._build_drawing()
    185 
    186     def save(self, filename):

~/Software/latticeproteins/latticeproteins/draw.py in _build_drawing(self)
    233         for y in range(self.shape[0]):
    234             for x in range(self.shape[1]):
--> 235                 self._add_item(x,y)
    236 
    237 class Drawing(svgwrite.Drawing):

~/Software/latticeproteins/latticeproteins/draw.py in _add_item(self, x, y)
    202         except KeyError:
    203             # Add color to specific letters if given
--> 204             color = COLORS[self.color_array[x][y]]
    205             self.drawing.letter(2*self.offset+self.font_size*x,
    206                 2*self.offset+self.font_size*y,

IndexError: list index out of range

```

**Comment by Zsailer** [comment]
Are you finding that it work for some sequences? i.e. does this only throw errors for some sequences, but not others?

**Comment by volkansevim** [comment]
Correct. If you keep generating random sequences, it eventually fails on
one of them.

**Comment by Zsailer** [comment]
Ok, thanks!

I'll investigate more this afternoon and let you know when it is fixed.

**Comment by Zsailer** [comment]
Ok, this problem should be fixed in PR #4.

**Comment by Zsailer** [comment]
Closing for now. Feel free to reopen if the problem continues on your machine.
